utils/constants.tsx

```diff
@@ -163,6 +163,7 @@ export const Preferences = {
 // For one off things that have a special, attention-grabbing UI until you interact with them
 export const Touched = {
     INVITE_MEMBERS: 'invite_members',
+    ADD_CHANNELS_CTA: 'add_channels_cta',
```

Comment:
:tada: glad `Touched` is getting more use, seemed like overkill when I added it and it was all by itself for months.